tests/src/core/testqgslayoutmap.cpp

```diff
@@ -626,13 +626,13 @@ void TestQgsLayoutMap::expressionContext()
   r = e6.evaluate( &c );
   QCOMPARE( r.toString(), QString( "longlat" ) );
 
-  QgsExpression e7( QStringLiteral( "@map_crs_pro4" ) );
-  r = e7.evaluate( &c );
-  QCOMPARE( r.toString(), QString( "+proj=longlat +datum=WGS84 +no_defs" ) );
+  //QgsExpression e7( QStringLiteral( "@map_crs_pro4" ) );
```

Comment:
The issue here is a type - "pro4", not "proj4"!

Reply:
I thought it would return an error when an item doesn't exist rather than an empty string. :/